aws/config.go

```diff
@@ -1,5 +1,10 @@
 package aws
 
+// UseServiceDefaultRetries instructs the config to use the service's own
+// default number of retries. This will be the default action if
+// Config.MaxRetries is nil also.
+const UseServiceDefaultRetries = -1
```

Comment:
This const isn't referenced anywhere, probably don't needed it. The V1 SDK used this value but the v2 doesn't since default retryer is created by the client.